### ISSUE TITLE
[Mod] Voicemutes are broken

### ISSUE THREAD

**Issue by Dav-Git** [Cog-Creators/Red-DiscordBot#4028]
# Other bugs

#### What were you trying to do?
Voice muting a user

#### What were you expecting to happen?
The user to get muted

#### What actually happened?
Modlog entry created. Nothing else happened

#### How can we reproduce this issue?
Voice mute someone.

Not sure if this is worth fixing as someone with manage channel perms can just mute people. If we don't want to fix it we might want to remove the voicemute command though

**Comment by yamikaitou** [comment]
Just tried to reproduce this and it worked fine for me. The Permission was successfully denied on the user I targeted.

**Comment by Vexed01** [comment]
It seems like this may be a Discord issue. Looking a the Discord permissions screen, when a user is already in a voice channel, it doesn't affect them unless the user rejoins. So this looks like a Discord issue. Maybe.

Edit - oh, and a modlog entry wasn't created for me.

**Comment by Jackenmen** [comment]
> It seems like this may be a Discord issue. Looking a the Discord permissions screen, when a user is already in a voice channel, it doesn't affect them unless the user rejoins. So this looks like a Discord issue. Maybe.

Yeah, it seems that way, and sadly I don't think we can do anything about it :/

> Edit - oh, and a modlog entry wasn't created for me.

Are you sure you have vmute case type enabled?

**Comment by Vexed01** [comment]
Ah, probably that. Except kicking and rejoining the user or using the API equivalent of ![http://i.vex.my.to/8G5QA](http://i.vex.my.to/8G5QAc) if such a thing exists as that button does it instantly.

**Comment by Jackenmen** [comment]
> Except kicking and rejoining the user

We can kick the users, but can't rejoin them.

> or using the API equivalent of [image of server mute checkbox] if such a thing exists as that button does it instantly.

And we can't use that as `[p]mute voice` is supposed to be a mute in single channel and this will do a voice mute for whole server (equivalent of `[p]voiceban`)

**Comment by Jackenmen** [comment]
Looking a little bit more into this, there's a *workaround* which requires Move Members permission - API allows you to "move" the user into the voice channel they're currently in, which makes the denied Speak permission actually affect that member. I don't know if it's intended, but as long as nobody in Discord says it isn't, we could just use that.

**Comment by Dav-Git** [comment]
If someone picks this up who is not me, I'd wait on #3634, maybe this can even be incorporated there @TrustyJAID ?

**Comment by Jackenmen** [comment]
> maybe this can even be incorporated there

It's out of scope for that PR, single PR should focus on as little as possible and that PR is already quite big while this issue requires a rather simple fix that does not require anything from #3634.
